TTS/tts/models/vits.py

```diff
@@ -717,7 +717,7 @@ def _freeze_layers(self):
             for param in self.text_encoder.parameters():
                 param.requires_grad = False
 
-            if hasattr(self, "emb_l"):
+            if hasattr(self, "emb_l") and self.emb_l is not None:
```

Comment:
When it is defined but set None?

Reply:
In init_multilingual it is defined as None if the model is not multilingual. In fact, the hasattr(self, "emb_l") is not needed given that it is None if multilingual approach is not used.